doc/content/integrations/pubsub/message-types/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Message Types"
+description: ""
+---
+
+The Application Server can be configured to **publish** messages for any of the following events:
+- Uplink Message
+- Join Accept
+- Downlink Ack
+- Downlink Nack
+- Downlink Sent
+- Downlink Failed
+- Downlink Queued
+- Location Solved
+- Downlink Push
+- Downlink Replace
+- Service Data
+
+The Application Server can be configured to **subscribe** to messages to schedule the following events:
+- Downlink Push
+- Downlink Receive
```

Comment:
Idem

Reply:
Are the topics Title Case or Sentence case?

Reply:
and do you mean sort alphabetically? so

Publish:
- Downlink ack
- Downlink failed
- Downlink nack
- Downlink queued
- Downlink sent
- Join accept
- Location solved
- Service data
- Uplink message

Publish
- Downlink queue push
- Downlink queue receive